--- config.h	2020-06-14 12:25:02.684064833 +0800
+++ config.h.bak	2020-06-14 12:27:19.550735565 +0800
@@ -5,8 +5,8 @@
  *
  * font: see http://freedesktop.org/software/fontconfig/fontconfig-user.html
  */
-static char *font = "mono:pixelsize=14:antialias=true:autohint=true";
-static char *font2[] = { "JoyPixels:pixelsize=10:antialias=true:autohint=true" };
+static char *font = "dejavu sans mono:pixelsize=16:antialias=true:autohint=true";
+static char *font2[] = { "Inconsolata for Powerline:pixelsize=14:antialias=true:autohint=true" };
 static int borderpx = 2;
 
 /*
